src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/form_fields.html.twig

```diff
@@ -0,0 +1,48 @@
+{# **
```

Comment:
btw @jolelievre we could rename this to something specific for suppliers I suppose

Reply:
I named it `form_fields` originally because I thought it could contain other form overrides for other parts But I guess you're right it's probably cleaner to have a dedicated file for each theme